package.json

```diff
@@ -202,18 +202,19 @@
     "@lingui/core": "3.15.0",
     "@lingui/cli": "3.15.0",
     "@lingui/macro": "3.15.0",
-    "@web3-onboard/core": "patch:@web3-onboard/core@npm%3A2.15.0#./.yarn/patches/@web3-onboard-core-npm-2.15.0-66f779b1b5.patch"
+    "@web3-onboard/core": "patch:@web3-onboard/core@npm%3A2.15.0#./.yarn/patches/@web3-onboard-core-npm-2.15.0-66f779b1b5.patch",
+    "electron-to-chromium": "1.4.311"
```

Comment:
why  this is needed ? you don't import / refer to this package in Your changes

Reply:
no not needed, was part of me trying to fix a babel issue
this was not part of the fix, so removed now